### ISSUE TITLE
should we discourage GitHub comments via email?

### ISSUE THREAD

**Issue by jamiebresch** [wrf-model/WRF#1649]
My personal preference is to comment on GitHub issues/pull requests directly on the GitHub web site, so the messages stay 'cleaner' to view in both web pages and email notifications.

**Comment by davegill** [comment]
@jamiebresch @weiwangncar @kkeene44 @dudhia @smileMchen @jordanschnell @liujake @cenlinhe 
I completely agree. Take a look at #1609.
It is indecipherable!

**Comment by weiwangncar** [comment]
I do like that idea. But how do you discourage people from doing that...

**Comment by cenlinhe** [comment]
I agree too about commenting directly on the Github page instead of email

**Comment by mgduda** [comment]
I'll add my name to the list of people who think that email comments quickly get to be unreadable. [Others seem to agree](https://github.com/dear-github/dear-github/issues/67), but as with other requests for GitHub capabilities, I have my doubts that GitHub will ever implement the ability to turn off email comments.

**Comment by XuxaHV** [comment]
someone who can help me to use the model for an investigation please?

**Comment by kkeene44** [comment]
I also agree.
